### DISCUSSION TITLE
Design Ideen

### DISCUSSION THREAD

**Discussion by laralngn** [SE-TINF22B2/G2-Schoolify#12] [Ideas]
Hier ein paar Templates an denen wir uns beim Design orientieren könnten (z.B. Komponenten, Aufbau, Farbkonzepte).

### Components
[Components](https://www.figma.com/file/ItcNsvnNun4o89treHyWKf/%40shadcn%2Fui---Design-System-(Community)?type=design&node-id=4-6598&mode=design&t=qjupnAFsVMMheVxZ-0)

### Dashboards
- [Dashboard Academic Management UI Design](https://www.figma.com/community/file/1286964964654485066/dashboard-academic-management-ui-design) ⭐️
- [Dashboard UI Kit - Dashboard](https://www.figma.com/community/file/1210542873091115123/dashboard-ui-kit-dashboard)
- [Dashboards UI Kit](https://www.figma.com/community/file/1135477961222225742/dashboards-ui-kit) ⭐️
- [School Management Admin Dashboard UI](https://www.figma.com/community/file/1219642652767985298)
- [School Admission Dashboard](https://www.figma.com/community/file/1096776247407806975) ⭐️
- [SAAS base school/ Coaching automation platform](https://www.figma.com/community/file/1236656143906489902)
- [Bank Management Dashboard](https://www.figma.com/community/file/1150832956726214755/bank-management-dashboard)
- [Unab/MiportalU](https://www.figma.com/community/file/1076318523555847759/unab-miportalu)
- [Purity UI Dashboard - Chakra UI Dashboard](https://www.figma.com/community/file/1020707462188017225/purity-ui-dashboard-chakra-ui-dashboard)
- [Dashboards & UI kit for Artificial Intelligence – Quickit AI](https://www.figma.com/community/file/1249338350862747811/dashboards-ui-kit-for-artificial-intelligence-quickit-ai) ⭐️

### Websites
- [SmartAcademy-Kids Education Kindergarten](https://www.figma.com/community/file/1291938383007998890)
- [Elementary School Website Templates](https://www.figma.com/community/file/1112093023471943331/elementary-school-website-templates)
- [School website](https://www.figma.com/community/file/1105069539354994759)
- [University UI KIT](https://www.figma.com/community/file/1193461935826510859/university-ui-kit-by-magika)

**Comment by laralngn** [comment]
Habe meine Favoriten mit ⭐️ markiert. Zum Teil wegen des Designs, zum Teil wegen Komponenten / Aufbau.

**Comment by Schmarvinius** [comment]
Hier sind alle Seiten die erstellt werden müssen: 
(Bei Ergänzungen einfach diesen Kommentar ergänzen)


#### Dashboard als Homepage 
- Kalender
- Stundenplan
- Mensaplan
- Api-Seiten
- Sign-in
- Einstellungen
- News

#### Adminseiten
- Verwaltung Seite (Übersicht über alle Admin Funktionen)
- Neue Schüler hinzufügen
- Neue Lehrer hinzufügen
- Stundenplan erstellen
- Krankmeldungen einsehen
- Übersicht über alle Klassenbücher
- Einsicht aller Klassen + Schüler (+Schüler Ansicht)

#### Lehrerseiten
- Klassenbuch
- Editierbarer Stundenplan (Ausfall, Vertretung)
- Notenvergabe

#### Schülerseiten
- Noteneinsicht
- Notifications
- Krankmeldung

**Reply by laralngn** [reply]
Sind nicht fast alle Sachen die beim Dashboard stehen auch eigene Seiten?
